inputs/homologous.pin

```diff
@@ -0,0 +1,146 @@
+     	    g.n.coord, &
+	    g.c.alpha, &
+	    g.c.detgam, &
+	    g.c.gcov, &
+	    g.c.dg, &
+	    g.c.dalpha, &
+	    flux_divergence, &
+	    src_terms
+
+file_type   = hdf5      # Tabular data dump
+dt          = 5         # time increment between outputs
+
+<parthenon/output2>
+file_type = hst
+dt = 1
+
+<parthenon/time>
+nlim        = -1        # cycle limit
+tlim        = 5000      # time limit
+integrator  = rk2       # time integration algorithm
+ncycle_out  = 1         # interval for stdout summary info
+dt_init_fact = 1.e-6
+
+<parthenon/mesh>
+nghost = 4
+#refinement = adaptive
+#numlevel = 3
+
+nx1         = 1024        # Number of zones in X1-direction
+x1min       = 0           # minimum value of X1
+x1max       = 250         # maximum value of X1
+ix1_bc      = user        # Inner-X1 boundary condition flag
+ox1_bc      = user        # Outer-X1 boundary condition flag
+
+nx2         = 1             # Number of zones in X2-direction
+x2min       = 0             # minimum value of X2
+x2max       = 3.14159265359 # maximum value of X2. Pi
+ix2_bc      = user          # Inner-X2 boundary condition flag
+ox2_bc      = user          # Outer-X2 boundary condition flag
+
+nx3         = 1             # Number of zones in X3-direction
+x3min       = 0             # minimum value of X3
+x3max       = 6.28318530718 # maximum value of X3. 2*pi
+ix3_bc      = periodic      # Inner-X3 boundary condition flag
+ox3_bc      = periodic      # Outer-X3 boundary condition flfgag
+
+num_threads = 1         # maximum number of OMP threads
+
+<parthenon/meshblock>
+nx1 = 1024
+nx2 = 1
+nx3 = 1
+
+<parthenon/refinement0>
+field = c.c.bulk.rho
+method = derivative_order_1
+max_level = 3
+
+<eos>
+type = IdealGas
+Gamma = 1.33334
+Cv = 1.0
+
+<physics>
+hydro = true
+he = false
+3t = false
+rad = false
+
+<fluid>
+xorder = 2
+cfl = 0.25
+riemann = llf
+recon = weno5
+c2p_max_iter = 1000
+Ye = false
+
+<fixup>
+enable_floors = true
+enable_ceilings = true
+enable_flux_fixup = true
+enable_c2p_fixup = true
+floor_type = X1RhoSie
+rho0_floor = 1.0e-8
+sie0_floor = 1.0e-3
+rho_exp_floor = -2.0
+sie_exp_floor = -1.0
+ceiling_type = ConstantGamSie
+sie0_ceiling = 10.0;
+gam0_ceiling = 10.0;
+
+<monopole_gr>
+enabled = true
+npoints = 2048
+rout = 250
+# Disables time derivatives. Runs only at initialization
+force_static = false
+# Runs the first n subcycles. Then freezes. Source terms are not disabled.
+# -1 means it's always run.
+run_n_times = -1
+# dtfac is for time step control. 0 < dtfac <= 1.
+# time step will be <= dtfac*(time rate of change of lapse)
+dtfac = 0.9
+# warn_on_dt tries to estimate if first-order operator split is sufficiently accurate.
+# Compares (dalpha/dt) computed analytically with numerical difference
+# between two subcycles. Warns if difference is larger than dtwarn_eps
+warn_on_dt = false
+dtwarn_eps = 1e-5
+
+<homologous>
+table="/home/mari/Work/Phoebus/phoebus/Tests/Reader/converted_ADM_homologouscollapse.dat"
```

Comment:
Try to make this a relative path

Reply:
changed to "../Tests/Reader/converted_ADM_homologouscollapse.dat"